kuma/static/styles/components/newsletter.scss

```diff
@@ -61,13 +61,13 @@ input.newsletter-input-email {
     position: relative;
     width: 100%;
     padding: ($grid-spacing / 2);
-    @include bidi(((padding-right, 2em, padding-left, inherit),));
-    text-align: left;
+    @include bidi(((text-align, left, right),));
```

Comment:
I'm assuming that the padding you deleted has been replaced by the arrow positioning stuff below?